include/dmlc/base.h

```diff
@@ -11,6 +11,12 @@
 #define DMLC_USE_GLOG 0
 #endif
 
+/*!
+ * \def DMLC_USE_LOGGING_LIBRARY
```

Comment:
seems will cause doxygen warning issue, consider update

Reply:
Not sure the best way to do this.
```
#ifndef DMLC_USE_LOGGING_LIBRARY
#define DMLC_USE_LOGGING_LIBRARY
#undef DMLC_USE_LOGGING_LIBRARY
#endif
```

Does that seem reasonable?

Reply:
How about we leave out the doxygen \def macro for now and make a generic comment here. Alternatively, comment about this option in the \file part of the logging.h